app/main.py

```diff
@@ -1,2 +1,96 @@
+from typing import Any, Hashable
+
+
+class Node:
+    def __init__(self, key: Hashable, hash_key: int, value: Any) -> None:
+        self.key = key
+        self.hash_key = hash_key
+        self.value = value
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            initial_capacity: int = 8,
+            load_factor: float = 0.75
+    ) -> None:
+        self._capacity = initial_capacity
+        self._load_factor = load_factor
+        self._size = 0
+        self._buckets = [[] for _ in range(self._capacity)]
```

Comment:
your hash table is actually a list of Nodes - mark this with annotation
Also filling up hash table with empty lists is not very efficient - better to fill out with None values